cves/CVE-2017-14056.yml

```diff
@@ -144,10 +155,12 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: The file is unit tested
```

Comment:
While it is unit tested, you do say in the fix that FATE doesn't hit this region. I'd explain that in `code_answer` as well, as that helps us explain how this vulnerability existed for so long without being noticed.